manifests/base/repo-server/argocd-repo-server-deployment.yaml

```diff
@@ -320,6 +320,25 @@ spec:
         volumeMounts:
         - mountPath: /var/run/argocd
           name: var-files
+      - command:
+          - /bin/cp
+          - -n
+          - /usr/local/bin/argocd
+          - /var/run/argocd/argocd
```

Comment:
Would it be microoptimization to link instead of copy?

Reply:
Changes made